python/lsst/ts/observatory/control/auxtel/atcs.py

```diff
@@ -628,6 +727,66 @@ async def close_dome(self):
                 f"{ATDome.ShutterDoorState.CLOSED}"
             )
 
+    async def _wait_for_shutter_door_state(self, state, cmd_task=None, timeout=None):
+        """Waits for `ATDome.ShutterDoorState.state` to match a required
```

Comment:
Please use an active voice for doc strings, i.e. `Wait for` instead of `Waits for`.